presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -3100,4 +3106,9 @@ public static boolean isInlineProjectionsOnValues(Session session)
     {
         return session.getSystemProperty(INLINE_PROJECTIONS_ON_VALUES, Boolean.class);
     }
+
+    public static int getMinColumnarEncodingChannelsToPreferRowWiseEncoding(Session session)
+    {
+        return session.getSystemProperty(MIN_COLUMNAR_ENCODING_CHANNELS_TO_PREFER_ROW_WISE_ENCODING, Integer.class);
```

Comment:
What's the default value for this? How to disable this? by setting it to an extreme large value? thanks!

Reply:
> What's the default value for this?

1000 columnar streams. For example VARCHAR - 3 streams (nulls, values, offsets). BIGINT - 2 streams (nulls, values. ARRAY<VARCHAR> - 5 streams, etc.

> How to disable this? by setting it to an extreme large value?

Yes